monorepo-folders/package.json

```diff
@@ -5,7 +5,7 @@
     "packages/*"
   ],
   "scripts": {
-    "start": "yarn build.workflows & concurrently --kill-others-on-fail -n 'frontend,api-server,worker' \"yarn frontend\"  \"yarn backend\" \"yarn worker\"",
+    "start": "DANGEROUSLY_DISABLE_HOST_CHECK=true yarn build.workflows & concurrently --kill-others-on-fail -n 'frontend,api-server,worker' \"yarn frontend\"  \"yarn backend\" \"yarn worker\"",
```

Comment:
What does this do?

Reply:
Allows cross-origin requests when using the webpack dev server. Gitpod runs them on different hostnames. I'll move it to the frontend package since it's only relevant for it.